### DISCUSSION TITLE
Inserting Custom Ascii Art

### DISCUSSION THREAD

**Discussion by lycheeje11y** [goolord/alpha-nvim#285] [Q&A]
I want to use custom ascii art for the dashboard.section.header value. Is there a way to insert ascii art that takes less work than copying it line by line into an array? Like referencing a txt file?

**Comment by gambhirsharma** [comment]
You can program a Lua function to parse all the lines of ASCII art from a txt file into the alpah-lua. I just chatGPT and found this code and it works too. 

```lua
local function read_ascii_art(file_path)
    local lines = {}
    for line in io.lines(file_path) do
        table.insert(lines, line)
    end
    return lines
end
local file_path = "./ASCII.txt"
local ascii_art_lines = read_ascii_art(file_path)
dashboard.section.header.val = ascii_art_lines
```
